tensorflow_privacy/privacy/membership_inference_attack/membership_inference_attack.py

```diff
@@ -96,6 +96,19 @@ def _run_threshold_attack(attack_input: AttackInputData):
       attack_type=AttackType.THRESHOLD_ATTACK,
       roc_curve=roc_curve)
 
+def _run_threshold_entropy_attack(attack_input: AttackInputData):
```

Comment:
Can we add an end-to-end test for this to membership_inference_attack_test.py? Something like the "test_run_attack_threshold_calculates_correct_auc" test

Reply:
The test case is added!